src/m365/spo/commands/page/Page.ts

```diff
@@ -105,10 +101,6 @@ export class Page {
       request
         .post(requestOptions)
         .then((res: any) => {
-          if (verbose) {
-            logger.log(res);
```

Comment:
Rather than removing these statements, we should change them to `logger.logToStderr`

Reply:
Same as the one above